Example/CryptoOnramp Example/CryptoOnramp Example/CryptoOnrampExampleView.swift

```diff
@@ -16,8 +16,9 @@ import StripePaymentSheet
 
 /// The main content view of the example CryptoOnramp app.
 struct CryptoOnrampExampleView: View {
+    @StateObject private var flowCoordinator = CryptoOnrampFlowCoordinator()
```

Comment:
This should’ve always been a `@StateObject` to work properly with `ObservableObject` / `@Published`. It wasn’t until adding the wallet selection view that we were getting lucky that other state was changing, allowing the `path` updates to propagate.